src/content/blog/maintainer-job-description.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "We're Recruiting DDEV Open Source Project Maintainers"
+pubDate: 2023-05-21
+summary: The DDEV community is seeking paid maintainers to boost the project's sustainability and growth.
+author: Randy Fay
+featureImage:
+    src: /img/blog/2023/05/diverse_group_of_people_sitting_at_a_table_working.png
+    alt: Image of a diverse group of maintainers sitting at a table maintaining DDEV
+categories:
+  - Announcements
+---
+
+We're recruiting DDEV Open Source Project Maintainers, both paid and volunteer positions! If you've ever considered becoming a more-involved contributor/maintainer on the project you love, we'd love to talk with you. DDEV needs to become more sustainable, with a larger group of maintainers, and we want to grow into that.
+
+### Maintainer Job Description
```

Comment:
This should either be level 2, or all `h3`’s should be `h2`’s. (No page should skip from 1 → 3.)

```suggestion
## Maintainer Job Description
```